docs/src/man/getting_started.md

```diff
@@ -558,6 +558,21 @@ Equivalently, the `in` function can be called with a single argument to create
 a function object that tests whether each value belongs to the subset
 (partial application of `in`): `df[in([1, 5, 601]).(df.A), :]`.
 
+!!! note
+
+    While `DataFrames` support indexing syntax that is very similar to
+    matrices, subsetting from a `DataFrame` will return a copy, not
+    a view.
+
+    `DataFrames` will only return a view when:
+
+    - a `!` is placed in the first indexing position (`df[!,:A]`, or `df[!, [:A, :B]]`),
+    - when using `.` notation (`df.A`),
+    - when a single row is selected using a single integer (`df[1, :A]`), or
+    - when the `@view` macro is used (`@view df[1:3, :A]`).
+
+    More details can be found in [`getindex` and `view`](@ref).  
```

Comment:
This link will not be correct. I would rather direct a reader to the section in the manual I have shared with you.

Reply:
Oh, ok -- sorry, I thought that was a link to that section (when I build locally links don't seem to work, so kinda in the dark...).

Reply:
yes - documenter is tricky. I think that the easiest thing is just to copy paste the physical link to master on GitHub (this is not ideal, but I assume we are not going to change this functionality in the near future so it should be good enough to do it this way).

Reply:
OK, thanks.